### PR TITLE
[Sprint 10 / Feature] Connect Akushon to Akushon App, Change Pose Name Config Usage in Action Manager

### PR DESCRIPTION
**Pull request by finesaaa**
## Jira Link: 

https://ichiro-its.atlassian.net/browse/PI-128
https://ichiro-its.atlassian.net/browse/PI-169

## Description

- Change pose name config usage in Action Manager
- Create get actions service to get all action data from akushon (read JSON data)
- Create set actions service to save all action data already change to akushon (write JSON file)
- Create run action service to play an action

Describe problems, if any, clearly and concisely.
Describe any changes that have been made in this pull request.

## Type of Change

- [ ] Bugfix
- [ ] Enhancement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause the existing functionality to not work as expected)

## How Has This Been Tested?

- [ ] New unit tests added.
- [x] Manual tested.

## Checklist:

- [x] Using Branch Name Convention
    - `feature/JIRA-ID-SHORT-DESCRIPTION` if has a JIRA ticket
    - `enhancement/SHORT-DESCRIPTION` if has/has no JIRA ticket and contain enhancement
    - `hotfix/SHORT-DESCRIPTION` if the change doesn't need to be tested (urgent)
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have made the documentation for the corresponding changes.

### CODE REVIEW

**Review by segara2410 — COMMENTED** [review]
since we already created new service to run an action, it is better to remove the action server one, just to prevent redundancy

**Review by segara2410 — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "akushon",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {}
+}
```
Comment (threeal) [inline package-lock.json:6]: what is this?

File: src/akushon/action/node/action_manager.cpp [hunk 2]
```diff
@@ -141,6 +151,15 @@ void ActionManager::start(int action_id, const Pose & initial_pose)
   is_running = true;
 }
 
+void ActionManager::start(const Action & action, const Pose & initial_pose)
+{
+  std::vector<Action> target_actions;
+  target_actions.push_back(action);
```
Comment (threeal) [inline src/akushon/action/node/action_manager.cpp:157]: better 👍 
```c++
std::vector<Action> target_actions {action};
```

**Review by threeal — APPROVED** [review]
the rest looks good though

**Pull request merged**